tests/test_clock.py

```diff
@@ -43,46 +43,42 @@ def test_timer_duration_exceeded(self):
             datetime.datetime(
-                2016, 7, 23, 10, 51, 9, 928000, tzinfo=pytz.utc),
-            datetime.datetime(
-                2016, 7, 27, 10, 51, 9, 928000, tzinfo=pytz.utc)
+                2000, 1, 1, 0, 0, 0, 0, tzinfo=pytz.utc), datetime.datetime(
+                    2000, 1, 5, 0, 0, 0, 0, tzinfo=pytz.utc)
         ]
         self.assertTrue(clock.Timer(self.mock_clock, duration).expired())
 
     def test_timer_duration_matched(self):
         duration = datetime.timedelta(hours=3 * 24)
         self.mock_clock.now.side_effect = [
             datetime.datetime(
-                2016, 7, 23, 10, 51, 9, 928000, tzinfo=pytz.utc),
-            datetime.datetime(
-                2016, 7, 26, 10, 51, 9, 928000, tzinfo=pytz.utc)
+                2000, 1, 1, 0, 0, 0, 0, tzinfo=pytz.utc), datetime.datetime(
+                    2000, 1, 4, 0, 0, 0, 0, tzinfo=pytz.utc)
         ]
-        self.assertFalse(clock.Timer(self.mock_clock, duration).expired())
+        self.assertTrue(clock.Timer(self.mock_clock, duration).expired())
 
     def test_timer_duration_not_matched_or_exceeded(self):
         duration = datetime.timedelta(hours=3 * 24)
         self.mock_clock.now.side_effect = [
             datetime.datetime(
-                2016, 7, 23, 10, 51, 9, 928000, tzinfo=pytz.utc),
-            datetime.datetime(
-                2016, 7, 24, 10, 51, 9, 928000, tzinfo=pytz.utc)
+                2000, 1, 1, 0, 0, 0, 0, tzinfo=pytz.utc), datetime.datetime(
+                    2000, 1, 3, 0, 0, 0, 0, tzinfo=pytz.utc)
         ]
         self.assertFalse(clock.Timer(self.mock_clock, duration).expired())
 
     def test_reset(self):
+        duration = datetime.timedelta(hours=3 * 24)
         self.mock_clock.now.side_effect = [
             datetime.datetime(
-                2016, 7, 23, 10, 51, 9, 928000, tzinfo=pytz.utc),
+                2000, 1, 1, 0, 0, 0, 0, tzinfo=pytz.utc), datetime.datetime(
+                    2000, 1, 2, 0, 0, 0, 0, tzinfo=pytz.utc), datetime.datetime(
+                        2000, 1, 4, 23, 59, 59, 999999, tzinfo=pytz.utc),
             datetime.datetime(
-                2016, 7, 26, 10, 51, 9, 928000, tzinfo=pytz.utc)
+                2000, 1, 5, 0, 0, 0, 0, tzinfo=pytz.utc)
         ]
-        timer = clock.Timer(self.mock_clock, None)
-        self.assertEqual(
-            timer._start_time,
-            datetime.datetime(
-                2016, 7, 23, 10, 51, 9, 928000, tzinfo=pytz.utc))
+        timer = clock.Timer(self.mock_clock, duration)
         timer.reset()
-        self.assertEqual(
-            timer._start_time,
-            datetime.datetime(
-                2016, 7, 26, 10, 51, 9, 928000, tzinfo=pytz.utc))
\ No newline at end of file
+        # First expired call is one microsecond before when the timer should
+        # expire, second is right when it should expire.
+        self.assertFalse(timer.expired())
+        self.assertTrue(timer.expired())
```

Comment:
I feel like by queueing up the `datetime.datetime` values, we're making it harder for the reader to understand the test. Because we start with all these times, but then it's on the reader to figure out how the times map to each line in the test, which is kind of hard.

Consider rewriting so that it's more obvious when the time changes are happening, e.g.

```python
    def test_reset(self):
        duration = datetime.timedelta(hours=3 * 24)
        self.mock_clock.now.return_value = datetime.datetime(
            2000, 1, 1, 0, 0, 0, 0, tzinfo=pytz.utc)
        timer = clock.Timer(self.mock_clock, duration)
        self.mock_clock.now.return_value = datetime.datetime(
            2000, 1, 2, 0, 0, 0, 0, tzinfo=pytz.utc)
        timer.reset()
        self.mock_clock.now.return_value = datetime.datetime(
            2000, 1, 4, 23, 59, 59, 999999, tzinfo=pytz.utc)
        # First expired call is one microsecond before when the timer should
        # expire, second is right when it should expire.
        self.assertFalse(timer.expired())
        self.mock_clock.now.return_value = datetime.datetime(
            2000, 1, 5, 0, 0, 0, 0, tzinfo=pytz.utc)
        self.assertTrue(timer.expired())
```